internal/client/valve/replay.go

```diff
@@ -27,6 +29,11 @@ func NewReplay(httpClient *http.Client) *Replay {
 	}
 }
 
+// NewDefaultReplay creates a Replay with default params.
+func NewDefaultReplay() *Replay {
+	return NewReplay(http.DefaultClient)
+}
+
```

Comment:
Can I know the reason behind creating `NewDefaultReplay()` instead of using `NewReplay()` and passing the `httpClient` value as `http.DefaultClient`?

Reply:
answered in NewDefaultAPI